openapi.yaml

```diff
@@ -2991,13 +3005,28 @@ components:
     forecast-rating-period:
       type: object
       additionalProperties: false
+      description: |
+        Forecast ratings data for a segment and period.  Includes status of the 
+        proposal relative to this data, as well as rating values.  
+        It also optionally may include a set of "branch-only" ratings.  
+        While forecast ratings in the "values" fields should be based on
+        ambient conditions, they may also be limited by current topology.  
+        The ratings provider should take this into account when populating the 
+        values field for the proposal.  
+
+        However, since this is a forecast, the Transmission Provider may run various 
+        look-ahead functions that change this topology such that it is no longer a limiting factor.  
+        To accommodate these scenarios, a "branch-only" rating should be sent that does not factor in 
```

Comment:
Is the "branch-only" rating set a forecasted AAR for a particular alternate topology? How can a rating *not* consider topology?

Reply:
For context, this was a requirement that actually came from internal sources (the EMS team, Jonathan and Arvind) at GE.  Generally, you're expecting to get ratings from the TOs that accommodate things like substation topology that could affect the rating on the line.  Concretely, our TO customers are going to get the same logic you are that looks at the customer's current outage plans and adjusts the ratings accordingly.  

The idea here is to capture the rating without an outage plan.  Or, put another way, the rating that you get is a function of the ambient temperature effect on the conductor, adjusted by any topology conditions the TO is aware of.  The "branch only" rating is JUST the temperature effect on the conductor.

Reply:
Is it accurate to say that the "branch only" rating is for the "base case" topology?